### PR TITLE
[script.module.youtube_channels] 0.2.0

### PR DESCRIPTION
**Pull request by goggle**
### Description

Small script to support YouTube channels. This is already available in Krypton, but no longer maintained.
It is needed for `script.module.srgssr`.
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.module.youtube_channels/addon.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="plugin.video.youtube" version="6.8.18+matrix.1"/>
+    <import addon="script.module.simplecache" version="2.0.2"/>
+  </requires>
+  <extension point="xbmc.python.module" library="lib"/>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Helps you to include YouTube channels in a Kodi plugin.</summary>
+    <description lang="en_GB">YouTube Channels offers helper function to include a list of different YouTube channels into your Kodi plugin.</description>
+    <platform>all</platform>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
```
Comment (enen92) [inline script.module.youtube_channels/addon.xml:13]: please use SPDX license identifier instead of fullname (GPL-3.0-or-later)

**Review by enen92 — CHANGES_REQUESTED** [review]
Just a minor, then it's good to go

File: script.module.youtube_channels/addon.xml (hunk 1 above)
Comment (goggle) [reply script.module.youtube_channels/addon.xml]: Done.

**Comment by goggle** [comment]
Squashed the commits, any chances to get this merged?

**Review by enen92 — APPROVED** [review]

**Pull request merged**